src/js/tech/html5.js

```diff
@@ -728,32 +728,35 @@ class Html5 extends Tech {
       return super.addRemoteTextTrack(options);
     }
 
-    var track = document.createElement('track');
+    var htmlTrackElement = document.createElement('track');
 
     if (options['kind']) {
-      track['kind'] = options['kind'];
+      htmlTrackElement['kind'] = options['kind'];
```

Comment:
While you're already here, want to change all the bracket notation to dot notation?
`htmlTrackEleemnt['kind'] --> htmlTrackElement.kind`